docs/kyma-environment-broker/03-03-runtime-operations.md

```diff
@@ -34,6 +34,9 @@ The provisioning process contains the following steps:
 
 >**NOTE:** The timeout for processing this operation is set to `24h`.
```

Comment:
```suggestion
The timeout for processing the whole provisioning operation is set to `24h`. The provisioning steps delegate resource creation to Reconciler. Since Reconciler does not constrain a number of retries in case of failed reconciliation, KEB introduces a [provisioning timeout for Reconciler](../../resources/kcp/charts/kyma-environment-broker/values.yaml#L49) that is set to `2h`.
```